### ISSUE TITLE
When an event is changed leave slips corresponding to it no longer apply

### ISSUE THREAD

**Issue by billyeh** [attendanceproject/djattendance#783]
Example: this pre-training GOW and Apostles' Teaching were switched.

https://github.com/attendanceproject/djattendance/blob/89e41094b4340b7bfd49bfb7d3449c8daf9b16de/libraries/react/scripts/selectors/selectors.js#L98-L106

Whether the leave slip applies depends on the date of the event.

**Comment by davidhtien** [comment]
Is this still an issue?

**Comment by yipingbenjaminliu** [comment]
yes. It will be address when the front end for schedule changes are built

**Comment by yipingbenjaminliu** [comment]
#1306
